var/spack/repos/builtin/packages/rocfft/package.py

```diff
@@ -96,7 +96,8 @@ class Rocfft(CMakePackage):
     depends_on("googletest@1.10.0:", type="test")
     depends_on("fftw@3.3.8:", type="test")
     depends_on("boost@1.64.0: +program_options", type="test")
-    depends_on("llvm-amdgpu +openmp", type="test")
+    depends_on("llvm-amdgpu", type="test")
+    depends_on("rocm-openmp-extras", type="test")
```

Comment:
Just like with rocBLAS, `depends_on("llvm-amdgpu +openmp", type="test")` was just to make OpenMP available in the tests. I would expect `depends_on("rocm-openmp-extras", type="test")` to be sufficient.